sdk/storage/inc/blobs/blob_options.hpp

```diff
@@ -197,6 +219,16 @@ namespace Azure { namespace Storage { namespace Blobs {
      * are applied to every retrial.
      */
     std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
+
+    /**
+     * @brief Sets the customer provided key used when making requests.
```

Comment:
Sets or "holds"? This is a field so it doesn't set anything; it holds what should be used, right?

Reply:
fixed